packages/mongodb-chatbot-eval/src/withConfig.ts

```diff
@@ -92,50 +95,6 @@ export const withConfigOptions = <T>(
   });
 };
 // NOTE: consider moving the generic configurable CLI stuff to `mongodb-rag-core` or another independent library since we're using across multiple projects.
-/**
```

Comment:
Can't comment directly - should we remove the above note? We already have a ticket to refactor this into a separate package https://jira.mongodb.org/browse/EAI-251